tests-functional/clients/status_backend.py

```diff
@@ -163,7 +163,8 @@ def verify_is_valid_api_response(self, response):
             error = response.json()["error"]
             assert not error, f"Error: {error}"
         except json.JSONDecodeError:
-            raise AssertionError(f"Invalid JSON in response: {response.content}")
+            # Some responses may not be JSON
+            pass
```

Comment:
Just use `api_request` in this case for now. Like in `get_connection_string_for_bootstrapping_another_device`.

I agree that the functions naming should be improved, but now the tests are implemented with an assumption that `verify_is_valid_api_response` checks the JSON structure. We shouldn't change it I think.